### ISSUE TITLE
RpcDispatcher does not document user exceptions thrown of dispatchOncRpcCall

### ISSUE THREAD

**Issue by radai-rosenblatt** [dCache/oncrpc4j#17]
```java
try {
   program.dispatchOncRpcCall(call);
} catch (RpcException e) {
   call.reject(e.getStatus(), e.getRpcReply());
   _log.warn("Failed to process RPC request: {}", e.getMessage());
} catch (OncRpcException e) {
   call.failRpcGarbage();
   _log.warn("Failed to process RPC request: {}", e.getMessage());
} catch (IOException e) {
   call.failRpcGarbage();
   _log.warn("Failed to process RPC request: {}", e.getMessage());
}
```

any user exceptions (like NPEs) thrown out of dispatchOncRpcCall() are not caught, and not logged.
this makes tracking down bugs in user code more difficult

**Comment by kofemann** [comment]
what about this fix: 46b3d65

**Comment by radai-rosenblatt** [comment]
(copied from commit comment)
awesome. in theory things like scala can sneakily throw checked Exceptions without declaring them, but this si robust enough

**Comment by kofemann** [comment]
fixed in master 8f032f6
